test/AWS.jl

```diff
@@ -335,7 +365,7 @@ end
 
     @testset "PUT - w/ Params" begin
         body = "sample-file-body"
-        AWSServices.s3("PUT", "/$bucket_name/$file_name"; Body=body)
+        AWSServices.s3("PUT", "/$bucket_name/$file_name", Dict("body"=>body))
```

Comment:
I'm not sure what the convention in this package is but I usually put spaces around `=>`
```suggestion
        AWSServices.s3("PUT", "/$bucket_name/$file_name", Dict("body" => body))
```